installing/acs-installation-platforms.adoc

```diff
@@ -0,0 +1,73 @@
+:_content-type: ASSEMBLY
+[id="installation-platforms"]
+= Installation Platforms and Methods
+include::modules/common-attributes.adoc[]
+:context: installation-platforms
+
+toc::[]
+
+[role="_abstract"]
+{product-title} is supported on various platforms.
+This topic provides information for each platform and links to installation documentation.
+
+== Installation methods for different platforms
+
+You can perform different types of installations on different platforms.
+
+[NOTE]
+====
+Not all installation options are supported for all platforms, as shown in the following tables. Red Hat recommends that you do not use the `roxctl` install method unless you have a specific installation need that requires using this method.
+====
+
+.Self-managed platforms
+|===
+|Platform | Supported installation methods
+
+|Red Hat OpenShift Container Platform (OCP) 4.x a|
+* xref:../installing/install-ocp-operator.adoc#install-ocp-operator[Operator] (recommended)
+* xref:../installing/installing_helm/install-helm-quick.adoc#acs-install-helm-quick[Helm]
+* xref:../installing/install-quick-roxctl.adoc#install-quick-roxctl[roxctl]
```

Comment:
Maybe mark up "roxctl" in the link? If so, throughout the tables. There's also a few other instances in other files.